umbraco/Umbraco-CMS
# fixes #10297 8.14-RC - A few areas where focus lock doesn't work prop…

In all cases mentioned in the issue (https://github.com/umbraco/Umbraco-CMS/issues/10297), two infinite editors are closed at the same time which results in "editors.length" being counted wrong because of the slide animation.

We need to keep another count of open editors because the length of the editors' array is first changed when the animations are done.

How to test:
* Test the scenarios from the original issue and make sure you can use the interface when the infinite editors are closed.
